components/server/src/user/user-controller.ts

```diff
@@ -509,6 +511,7 @@ export class UserController {
 
         await this.userService.updateUserEnvVarsOnLogin(user, envVars);
         await this.userService.acceptCurrentTerms(user);
+        this.analytics.identify({ anonymousId: req.sessionID || "unknown", userId: user.id, });
```

Comment:
it would make sense to move it to `LoginCompletionHandler.complete`, which makes me think of the `request.login` call and if there is a chance to read the session id afterwards. 

L514 might already have a non-null req.sessionID, but this might change with `request.login`